docs/contributing.md

```diff
@@ -69,6 +75,6 @@ By making a contribution to this project, I certify that:
 -----------------------------------------------------------------------
 
 [OpenSBI main repository]: https://github.com/riscv/opensbi
+[OpenSBI mailing list]: mailto:opensbi@lists.infradead.org
```

Comment:
How does one subscribe to this mailing list? There's no mention of this list on http://lists.infradead.org/ or http://www.infradead.org/

Reply:
Updated PR with details about how to join mailing list.